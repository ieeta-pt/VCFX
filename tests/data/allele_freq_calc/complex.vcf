##fileformat=VCFv4.2
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read depth">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE1	SAMPLE2	SAMPLE3
1	100	rs1	A	G	30	PASS	AF=0.25	GT:AD:DP	0/1:10,5:15	0/0:20,0:20	1/1:0,25:25
1	200	rs2	C	T	40	PASS	AF=0.5	GT:AD:DP	0/1:8,7:15	0/1:10,10:20	0/0:18,2:20
