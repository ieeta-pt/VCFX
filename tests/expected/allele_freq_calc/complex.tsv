CHROM	POS	ID	REF	ALT	Allele_Frequency
1	100	rs1	A	G	0.5000
1	200	rs2	C	T	0.3333
